src/Stepping.TmProviders.LocalTm.Core/Stepping/TmProviders/LocalTm/Steps/ILocalTmStepExecutor.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Stepping.TmProviders.LocalTm.Steps;
+
+public interface ILocalTmStepExecutor
```

Comment:
"Executor" in the "Stepping.Core" project is only for executable steps.

To avoid ambiguity, can we create a `LocalTmProcessorBase` to replace this `ILocalTmStepExecutor`?